ci/jobs/slow-lint.sh

```diff
@@ -14,6 +14,9 @@ set -e
 echo -e "\n### Check tags on Bazel artifacts"
 ci/scripts/check-bazel-tags.sh
 
+echo -e "\n### Ensure bazel doesn't use 'git_repository's"
```

Comment:
Don't forget to update the script name here :)

Reply:
🫠